### ISSUE TITLE
Multiband Compressor Solo

### ISSUE THREAD

**Issue by p1839251** [sadko4u/lsp-plugins#21]
Hey

The solo button of the 'Multiband Compressor Mono/Stereo x8' doesn't work as expected.
If I solo a specific band, let's say the bass #0 band, I should only hear that band exclusively, only the very low frequencies, yet I hear some much higher frequencies as well. They are much quieter but still audible. 
This problem is true for all the other bands as well but this is the most noticeable one.
If you solo an inactive band, you shouldn't hear anything, should you?!

**Comment by sadko4u** [comment]
This is a result of implementation of the multiband compressor. It doesn't work like crossover in modern mode, so actually it's frequency curve consists of dynamic shelving filters instead of combination of static lowpass/hipass filters and amplifiers. This allows to avoid distortion of frequency characteristics at the split frequency. As the opposite effect, because we use shelving filters, we can not completely mute other bands. That's why all other bands are lowered by 36 dB. The behaviour of *solo* button is written in official documentation as such expected behaviour:
```
Controls - set of buttons that control the behaviour of the compressor:
    On - enables compressor assigned to the corresponding frequency band.
    S - turns on soloing mode to the selected band by applying -36 dB gain to non-soloing bands
    M - turns on muting mode to the selected band by applying -36 dB gain to it
```

**Comment by sadko4u** [comment]
Anyway, I think I should rework a bit filters to make their characteristics more sharp so we could better reduce the gain of non-soloing bands (for example, -48, -60 or -72  dB).

**Comment by sadko4u** [comment]
I've inspected the code. Seems that filters are working strange. I need to disover it...

**Comment by sadko4u** [comment]
The problem was in function that processes biquadratic filters and utilizes AVX+FMA3 instruction set.
This commit should fix the problem: d96927a1874c66ea9a639066597e101999b4eda4

**Comment by sadko4u** [comment]
@p1839251 are you available to build and test plugin on your system?

**Comment by sadko4u** [comment]
Merged changes into **devel** branch.
Will be available in the nearest release.

**Comment by sadko4u** [comment]
All changes are available in the 1.1.7 release that was just published.
